djangoproject/templates/aggregator/ecosystem.html

```diff
@@ -3,41 +3,78 @@
 
 {% block content %}
 
+  <h1>Community Ecosystem</h1>
```

Comment:
```suggestion
  <h1>Package Ecosystem</h1>
```
Should we align the page title to how it is described/linked on the index page:
https://github.com/django/djangoproject.com/blob/8d230390ceb35c892cc9110968412f42f8811f3c/djangoproject/templates/aggregator/index.html#L35

Reply:
I can add an `id` to this `h1` and an `a` with a `headerlink` class here if you want me to?

Reply:
Oh good thought 🤔 
Yes, I suppose that's similar to the docs and makes sense here also

Reply:
Done in cd596ff. ✔️